driver/base/response.go

```diff
@@ -29,9 +57,10 @@ type Response struct {
 	ElapsedTime        float64
 	FailedWhenContains []string
 	Failed             bool
```

Comment:
I wonder, shouldn’t we rather do ‘Failed OperationError’ and not a bool? In that case we won’t need the failedMsg as the Failed field would be of error itself with all the fields like message

Reply:
Yeah... so the thinking -- and totes open to changing/improving -- is to keep `Failed` for sorta compatibility with python scrapli, then replace where we call `Failed()` in kne to use ---> `OperationOk()` (which does return the new error containing the contextual info and such).

thoughts?

Reply:
ok, think I finally came around to what you're saying.... latest commit should be like you're saying I think.

we will need to make a one line update before next cmdo release to do a `r.Failed != nil` vs `!r.Failed` but otherwise I think -- despite lots of lines changed -- this shouldn't break anything :)

Reply:
Yes, looks like what I meant :)

Reply:
dope! will figure out what im doing w/ this and/or that other giant pr later this week and get it dialed in then make the update for kne! thanks dude!